webpack/ForemanTasks/Components/TasksTable/TasksTableSelectors.js

```diff
@@ -0,0 +1,44 @@
+import { translate as __ } from 'foremanReact/common/I18n';
+import { selectForemanTasks } from '../../ForemanTasksSelectors';
+
+export const selectTasksTable = state =>
+  selectForemanTasks(state).tasksTable || {};
+
+export const selectTasksTableContent = state =>
+  selectTasksTable(state).tasksTableContent || {};
+
+export const selectTasksTableQuery = state =>
+  selectTasksTable(state).tasksTableQuery || {};
+
+export const selectQuery = state => selectTasksTableQuery(state).query || {};
+
+export const selectLoading = state =>
+  selectTasksTableQuery(state).loading || false;
+
+export const selectPagitation = state =>
+  selectTasksTableQuery(state).pagination || {};
+
+export const selectItemCount = state =>
+  selectTasksTableQuery(state).itemCount || 0;
+
+const formatDate = date =>
+  date
+    ? new Date(date).toLocaleString('en-GB', {
+        hour12: false,
+        timeZoneName: 'short',
+      })
+    : __('N/A');
+
+const formatUsername = username => (username ? username.name : '');
+
+export const selectResults = state => {
+  const { results } = selectTasksTableContent(state);
+  if (!results) return [];
+  return results.map(result => ({
+    ...result,
+    username: formatUsername(result.username),
```

Comment:
Because you don't reuse `formatUsername` IMO it is redundant and you can just do:
```suggestion
    username: result.username ? result.username.name : '',
```